lvm2/pkg.yaml

```diff
@@ -41,58 +41,9 @@ steps:
              --enable-udev_sync \
              --enable-udev_rules \
              --enable-static_link \
+             --prefix=/usr \
              --with-udev-prefix=/usr \
-             MODPROBE_CMD=/sbin/modprobe
-
-        rm -f /sbin/blkdeactivate \
```

Comment:
not needed anymore?

Reply:
These aren't even under /rootfs. And yes, compiles just fine, no unnecessary stuff in the result